### ISSUE TITLE
How to prevent form submit with link prompt

### ISSUE THREAD

**Issue by patrickberkeley** [bustle/ember-mobiledoc-editor#2]
_I realize this project is WIP, so feel free to close this if the question will be resolved as the project progresses_

I'm using ember-content-kit 0.1.0 as follows:

```
    {{#content-kit-editor mobiledoc=mobileDoc on-change=(action 'onChange') as |contentKit|}}
        <button {{action contentKit.toggleMarkup 'strong'}}>B</button>
        <button {{action contentKit.toggleMarkup 'em'}}>I</button>
        <button {{action contentKit.toggleLink}}>Link</button>
      {{/content-kit-editor}}
```

One issue I'm running into is the `content-kit-link-prompt` does a full form submit which causes the page to refresh. How do I prevent that?

**Comment by mixonic** [comment]
Seems like a legit bug

**Comment by bantic** [comment]
closed by #132
